src/test/java/api/loans/DeclareClaimedReturnedItemAsMissingApiTests.java

```diff
@@ -0,0 +1,123 @@
+package api.loans;
+
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static api.support.matchers.LoanMatchers.hasLoanProperty;
+import static api.support.matchers.LoanMatchers.isClosed;
+import static api.support.matchers.ValidationErrorMatchers.hasErrorWith;
+import static api.support.matchers.ValidationErrorMatchers.hasMessage;
+import static api.support.matchers.ValidationErrorMatchers.hasParameter;
+import static org.folio.circulation.domain.representations.ChangeItemStatusRequest.COMMENT;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION;
+import static org.folio.circulation.domain.representations.LoanProperties.ACTION_COMMENT;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.ClaimItemReturnedRequestBuilder;
+import api.support.builders.DeclareClaimedReturnedItemAsMissingRequestBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+public class DeclareClaimedReturnedItemAsMissingApiTests extends APITests {
+  private InventoryItemResource item;
+  private String loanId;
+
+  @Before
+  public void setUpItemAndLoan() {
+    item = itemsFixture.basedUponSmallAngryPlanet();
+    loanId = loansFixture.checkOutByBarcode(item, usersFixture.charlotte())
+      .getId().toString();
+  }
+
+  @Test
+  public void canDeclareItemMissingWhenClaimedReturned() {
+    final String comment = "testing";
+
+    claimItemReturnedFixture.claimItemReturned(new ClaimItemReturnedRequestBuilder()
+      .forLoan(loanId)
+      .withItemClaimedReturnedDate(DateTime.now()));
+
+    claimItemReturnedFixture.declareClaimedReturnedItemAsMissing(
+      new DeclareClaimedReturnedItemAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment(comment));
+
+    assertLoanIsClosed(comment);
+    assertItemIsMissing();
+  }
+
+  @Test
+  public void cannotDeclareItemMissingWhenIsNotClaimedReturned() {
+    final Response response = claimItemReturnedFixture
+      .attemptDeclareClaimedReturnedItemAsMissing(new DeclareClaimedReturnedItemAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment("testing"));
+
+    assertThat(response.getStatusCode(), is(422));
+    assertThat(response.getJson(), hasErrorWith(hasMessage("Item is not Claimed returned")));
+    assertThat(response.getJson(), hasErrorWith(hasParameter("id", item.getId().toString())));
```

Comment:
I think this is intended to match a single validation error, with both a message and a parameter.

If it is, then I think the intent is clearer (and more specific) if using a form similar to:
```suggestion
    assertThat(response.getJson(), hasErrorWith(allOf(
      hasMessage("Item is not Claimed returned"),
      hasUUIDParameter("id", item.getId()))));
```

(I've also replaced the parameter check with a more specific UUID version)